db/migrate/20180219125251_add_campaign_id_to_requested_accounts.rb

```diff
@@ -0,0 +1,5 @@
+class AddCampaignIdToRequestedAccounts < ActiveRecord::Migration[5.1]
```

Comment:
Instead of this, you can just use a scope: Campaign has_many RequestedAccounts through Courses.